CMakeLists.txt

```diff
@@ -161,54 +154,6 @@ option(
   ON)
 option(ENABLE_AVX2 "Enable AVX2 optimizations" OFF)
 
-set(ACTIVE_NETWORK
```

Comment:
This is passed as a #define to the compiler so I think this needs to stay in the main cmake file.